### PR TITLE
Copying app-project into new app-project

### LINKED ISSUE #157: Migrate app-project
**Package** app-project

**Feature or Issue Description**
The app-project setup needs to be migrated to app-project-2 and then app-project-2 to be renamed. It easier to do the update to the next version of next.js this way since we weren't far in development. 

**To Dos**

- [ ] Example

### PR DESCRIPTION
**Pull request by srallen**
Package: app-project

Closes #157.

Describe your changes:
Copying over the project specific code and making updates as I go for the new version of next.js. Still WIP.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

**Comment by rogerhutchings** [comment]
Okay, the only thing left at this point is fixing the stores - for some reason, named imports in MST are playing up a bit, so I'm having to `import * as mst` and access them as properties of the `mst` object. I'll be able to finish this tomorrow.

**Comment by rogerhutchings** [comment]
Okay, this should now all be working as before, with the added bonus that data fetching and store provision has been moved into `_app`, so all pages should now have access once the store is injected into it.

**Comment by rogerhutchings** [comment]
@srallen this should be ready to go

File: packages/app-project/package.json [hunk 1]
```diff
@@ -2,53 +2,44 @@
...
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
-    "jsdom": "^11.8.0",
-    "mocha": "^5.1.1",
-    "sinon": "^4.5.0",
-    "standard": "^11.0.1",
-    "standard-loader": "^6.0.1"
-  },
-  "engines": {
-    "node": "8"
```
Comment (srallen) [inline packages/app-project/package.json]: check-engines doesn't work without a specified node engine version in package.json

**Pull request merged**